CONTRIBUTING.md

```diff
@@ -29,3 +29,78 @@ Recommendations and requirements for how to best contribute to Parquet. We striv
 ### License
 By contributing your code, you agree to license your contribution under the terms of the APLv2:
 https://github.com/apache/parquet-format/blob/master/LICENSE
+
+### Additions/Changes to the Format
+
+The general steps for adding features to the format are as follows:
+
+1. Discuss changes on on the developer mailing list (dev@parquet.apache.org).  Often times it is helpful to link to a draft pull request to make the discussion concrete. This step is complete when there lazy consensus.
+
+2. One a change has lazy consensus two implementations of the feature
+demonstrating interopability must also be provided.  One implementation MUST be [parquet-java](http://github.com/apache/parquet-java).  It is preferred that the second implementation be [parquet-cpp](https://github.com/apache/arrow) or [parquet-rs](https://github.com/apache/arrow-rs), however at the discretion of the PMC any
+open source Parquet implementation may be acceptable.
+
+3. After the first two steps are complete a formal vote is held on the Parquet mailing list to officially
+ratify the feature.  After the vote passes the format change is merged into the parquet-format repository
+and it is expected the change in step 2 will also be merged soon after.
```

Comment:
parquet-java would have to wait until the next release of parquet-format because it cannot depend on the SNAPSHOT version.

Reply:
This is a good point that I missed.  Thank you.  I'll update this to point out that a release of the format is required for merging into parquet-java.

Reply:
I indicated this in the instructions.  Given that this advocates releasing the format as features are added, hopefully this won't be a blocker?